### DISCUSSION TITLE
Increase width of ledger lines

### DISCUSSION THREAD

**Discussion by ghost** [helge17/tuxguitar#525] [General]
I'm sorry but for me those ledger lines are barely visible:

![image](https://github.com/user-attachments/assets/d4cc6db3-b502-419d-8106-fc7407a50fff)

How can I increase the width of the ledger lines?

**Comment by guiv42** [comment]
Better than increasing width, you can make the lines darker.
See menu Tools -> Settings, then select "Styles" and "Horizontal Lines Color"
![light](https://github.com/user-attachments/assets/c02ebad9-16b9-43b5-ad03-3752a8dc276c) ![black](https://github.com/user-attachments/assets/48118e50-46db-40f9-990f-215396ab812f)

**Reply by ghost** [reply]
ok that works
